server.go

```diff
@@ -510,22 +510,40 @@ func (s *Server) HandleTracePacket(packet []byte) {
 		return
 	}
 
-	sample, metrics, err := samplers.ParseSSF(packet)
+	msg, err := samplers.ParseSSF(packet)
 	if err != nil {
 		reason := fmt.Sprintf("reason:%s", err.Error())
 		s.Statsd.Count("packet.error_total", 1, []string{"packet_type:ssf_metric", reason}, 1.0)
 		log.WithError(err).Warn("ParseSSF")
 		return
 	}
-
+	metrics, err := msg.Metrics()
+	if err != nil {
+		reason := fmt.Sprintf("reason:%s", err.Error())
+		s.Statsd.Count("packet.error_total", 1, []string{
+			"packet_type:ssf_metric",
+			"step:extract_metrics",
+			reason,
+		}, 1.0)
+		log.WithError(err).Warn("ParseSSF-metrics")
```

Comment:
Can we be a little more descriptive with the message than "ParseSSF-metrics"? (So that someone tailing the logs doesn't have to reference the code to figure out whether it's relevant to them)